indra/newview/llappviewermacosx-objc.h

```diff
@@ -33,4 +33,6 @@
 //Why?  Because BOOL
 void launchApplication(const std::string* app_name, const std::vector<std::string>* args);
 
+void force_ns_sxeption() { [NSEXception raise:@"Forced NSException"] };
```

Comment:
```suggestion
void force_ns_sxeption() { [NSException raise:@"Forced NSException"] };
```
Mac build is unhappy. Probably due to typo.

Reply:
Thank you, that's why it's a draft, will run it on mac a bit later.